src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute in the link tag is pointing to a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files. You should compile the SCSS into a CSS file and link to that CSS file instead.